### ISSUE TITLE
Vuetify the voila tree

### ISSUE THREAD

**Issue by marketneutral** [QuantStack/voila-vuetify#14]
Filing an enhancement request ... would be very nice to have a Material Design tree page when you start `voila` in a directory without a single target notebook.

**Comment by maartenbreddels** [comment]
Good idea, not a high priority from our side at this moment, but for sure something we'd like to have.

**Comment by marketneutral** [comment]
In the meantime, I'm going to work on the following
- I put all the notebooks I want to serve into, say, `/srv/voila/notebooks` including a `homepage.ipynb` notebook, which I style with `ipyvuetify`
- I run `voila` in this directory
- my `homepage.ipynb` includes a Vuetify nice list or expansion panel, etc. populated with directory contents of `/srv/voila/notebooks` and meta-data (like "this notebook does X,Y,Z") pulled out of each notebook using [scrapbook](https://github.com/nteract/scrapbook)
- there is an icon for each item in the list where the href is `http://server:8866/voila/render/` + the notebook name.
- I give the users the URL `http://server:8866/voila/render/homepage.ipynb` as the root

**Comment by maartenbreddels** [comment]
That's a nice workaround, once you have that it should be relatively easy to create a tree.html from it.

**Comment by marketneutral** [comment]
I made a project for this. Would love any comments!

https://github.com/marketneutral/voila-homepage
